src/packages/formitem/formitem.taro.tsx

```
@@ -196,42 +208,48 @@ export class FormItem extends React.Component<
-        className={`nut-form-item ${className}`}
+        className={`${this.getClassNameWithDirection('nut-form-item')} ${className}`}
         style={style}
         align={align}
         onClick={(e) =>
           this.props.onClick && this.props.onClick(e as any, this.componentRef)
         }
       >
         {label ? (
-          <View className="nut-cell-title nut-form-item-label">
+          <View
+            className={`nut-cell-title ${this.getClassNameWithDirection('nut-form-item-label')}`}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)